### PR TITLE
feat: sorted set put value and fetch by index implementation

### PR DESCRIPTION
**Pull request by malandis**
Implements `sortedSetPutValue` and `sortedSetFetchByIndex`.

### CODE REVIEW

File: src/internal/cache-client.ts [hunk 1]
```diff
@@ -1640,6 +1643,172 @@ export class CacheClient {
     });
   }
 
+  public async sortedSetPutValue(
```
Comment (cprice404) [inline src/internal/cache-client.ts:1646]: for similarity with Sets we may wanna go with Element here.

File: src/internal/cache-client.ts [hunk 2]
```diff
@@ -1640,6 +1643,172 @@ export class CacheClient {
...
+    endIndex?: number
+  ): Promise<CacheSortedSetFetch.Response> {
+    try {
+      validateCacheName(cacheName);
+      validateSortedSetName(sortedSetName);
+    } catch (err) {
+      return new CacheSortedSetFetch.Error(normalizeSdkError(err as Error));
+    }
+    //this.logger.trace(
+    //`Issuing 'sortedSetFetchByIndex' request; startIndex: ${startIndex}, endIndex : ${endIndex}`
```
Comment (cprice404) [inline src/internal/cache-client.ts:1727]: can use the logger varargs signature here to avoid the linting error

**Review by cprice404 — APPROVED** [review]

File: src/internal/cache-client.ts (hunk 1 above)
Comment (malandis) [reply src/internal/cache-client.ts]: Fixed in https://github.com/momentohq/client-sdk-nodejs/pull/314/commits/3d63768766c96de67c2e103e8705369a6b28a6c2

**Review by cprice404 — APPROVED** [review]

File: src/internal/cache-client.ts (hunk 2 above)
Comment (malandis) [reply src/internal/cache-client.ts]: Fixed in [49daeee](https://github.com/momentohq/client-sdk-nodejs/pull/314/commits/49daeeef75b3f569d3db576b7bf01e5ff5ae2923)

**Review by cprice404 — DISMISSED** [review]

**Review by cprice404 — APPROVED** [review]

**Pull request merged**